dotCMS/src/main/java/com/liferay/portal/ejb/CompanyManagerUtil.java

```diff
@@ -196,12 +195,13 @@ public static void updateUsers(java.lang.String languageId,
 	throws PortalException, SystemException,com.dotmarketing.exception.DotRuntimeException {
 		try{		
 			CompanyManager companyManager = CompanyManagerFactory.getManager();
-			//companyManager.updateUsers(languageId, timeZoneId, skinId, dottedSkins, roundedSkins, resolution);
-			companyManager.updateDefaultUser(languageId, timeZoneId, skinId, dottedSkins, roundedSkins, resolution);
-			
-			
-			
-			TimeZone.setDefault(TimeZone.getTimeZone(timeZoneId));

```

Comment:
Do we still need this line?  I think we do, right?

Reply:
@wezell I think not. That happens at https://github.com/dotCMS/core/pull/20114/files#diff-a7359e9c2e1536f6d9fad70abb0e8d2262413aefe781c6df5822d25e7012e137R42.

Reply:
humm, what happens if the `timeZoneValid` return valid but the company/user fails to save for some reason? The server could get into a state whereby the server timezone and the company timezone don't match?  I think I like the `timeZoneValid` to not be the one to set the system timezone as it is intended to only be a test.

Reply:
ok do you mean we have to do a soft set, and rollback to the previous at the end, so if the soft set is good, we can proceed and set in whatever we want right
I guess if it is the case we can rename the method to isTimeZoneValid and set the time zone in the if

Reply:
You are right. Timezone set restored and also unsetting at `finally` code block and method has been renamed.